utils/debug/Debug.js

```diff
@@ -8,7 +8,7 @@ import View from '../../src/Core/View';
 
 function createChartContainer(chartDiv, width, left, chartId) {
     const div = document.createElement('div');
-    div.style = `position: absolute; bottom: 0; left: ${left}vw; width: ${width}vw; height: 20rem; background-color: white;`;
+    div.style.cssText = `position: absolute; bottom: 0; left: ${left}vw; width: ${width}vw; height: 20rem; background-color: white;`;
```

Comment:
Some webkit-based, like midori or Epiphany. Safari is or maybe was affected at some point too, if I believe comments on the web (I have no way of testing though)